.github/workflows/release.yaml

```diff
@@ -0,0 +1,91 @@
+# Copyright 2025 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: release-actions
+
+on:
+  schedule:
+   - cron: '0 0 * * 1' # weekly on Monday at 00:00
+
+  workflow_dispatch:
+    inputs:
+      dry_run:
+        description: "dry-run mode: if true, no git tags will be pushed."
+        type: boolean
+        default: false
+      release_type:
+        description: "Type of Release"
+        required: true
+        default: build
+        type: choice
+        options:
+          - major
+          - minor
+          - patch
+          - prerelease
+          - build
+      forced_version:
+        description: "(Optional) SemVer2-compliant forced-version to tag explicitly, instead of auto-bumping.
+                      Must not already exist"
+        required: false
+        type: string
+
+permissions: {}
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read # to read the repo and tags
+      id-token: write # to inject the OIDC token to octo-sts
+
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@c6295a65d1254861815972266d5933fd6e532bdf # v2.11.1
+        with:
+          egress-policy: audit
+
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Check if any changes since last tag
+        id: check
+        run: |
+          git fetch --tags
+          if [ -z "$(git tag --points-at HEAD)" ]; then
+            echo "Nothing points at HEAD, bump a new tag"
+            echo "bump=yes" >> $GITHUB_OUTPUT
+          else
+            echo "A tag already points to head, don't bump"
+            echo "bump=no" >> $GITHUB_OUTPUT
+          fi
+
+      - uses: octo-sts/action@6177b4481c00308b3839969c3eca88c96a91775f # v1.0.0
+        id: octo-sts
+        with:
+          scope: ${{ github.repository }}
+          identity: release
+
+      - name: Bump and push Git tag
+        uses: chainguard-dev/actions/git-tag@855d928df4d35ab87d1495993810ba7725a2f8bb # v1.0.0
```

Comment:
I guess you need a v1.0.1 tag to get this fix https://github.com/chainguard-dev/actions/pull/534?